### PR TITLE
Fix node push to public site

### PR DESCRIPTION
**Pull request by bianchim**
MULTISITE-12765

### CODE REVIEW

File: lib/features/farnet_communities/farnet_communities.module [hunk 1]
```diff
@@ -8,6 +8,151 @@
...
+ */
+function farnet_communities_push_form_admin($form, &$form_state) {
+  $group_content_types = og_get_all_group_content_bundle();
+
+  $form['farnet_communities_push_public_bundles'] = [
+    '#type' => 'select',
+    '#title' => t('Content types'),
+    '#multiple' => TRUE,
+    '#options' => $group_content_types['node'],
+    '#default_value' => variable_get('push_public_bundles', []),
```
Comment (gervasek) [inline lib/features/farnet_communities/farnet_communities.module]: Variable should be ```"farnet_communities_push_public_bundles"```

File: lib/features/farnet_communities/farnet_communities.module [hunk 2]
```diff
@@ -8,6 +8,151 @@
 include_once 'farnet_communities.features.inc';
 
 /**
+ * Implements hook_menu().
+ */
+function farnet_communities_menu() {
+  $items['node/%node/push'] = array(
+    'title' => 'Push node to public site',
```
Comment (gervasek) [inline lib/features/farnet_communities/farnet_communities.module]: I think we can remove "node" :
- the title is long for a menu tab
- the "node" term is not familiar for a simple Editor

File: lib/features/farnet_communities/farnet_communities.module [hunk 3]
```diff
@@ -8,6 +8,151 @@
...
+  return system_settings_form($form);
+}
+
+
+/**
+ * Push a node to the public site.
+ */
+function farnet_communities_push_form($form, &$form_state, $node) {
+  // test.
+  $form['confirm_text'] = [
```
Comment (gervasek) [inline lib/features/farnet_communities/farnet_communities.module]: Why you don't use confirm_form ? :P

File: lib/features/farnet_communities/farnet_communities.module [hunk 4]
```diff
@@ -8,6 +8,151 @@
...
+
+  return system_settings_form($form);
+}
+
+
+/**
+ * Push a node to the public site.
+ */
+function farnet_communities_push_form($form, &$form_state, $node) {
+  // test.
```
Comment (gervasek) [inline lib/features/farnet_communities/farnet_communities.module]: the comment ``` "// test"``` seems to be useless

File: lib/features/farnet_communities/farnet_communities.module [hunk 5]
```diff
@@ -8,6 +8,151 @@
...
+  }
+
+  // Check the node type.
+  $can_push = variable_get('farnet_communities_push_public_bundles', []);
+
+  if (!in_array($node->type, $can_push) || empty($node->og_group_ref)) {
+    return FALSE;
+  }
+
+  // Check the URL to be sure we're on the published node only.
```
Comment (gervasek) [inline lib/features/farnet_communities/farnet_communities.module]: I don''t understand this check...

File: lib/features/farnet_communities/farnet_communities.module (hunk 1 above)
Comment (gervasek) [inline lib/features/farnet_communities/farnet_communities.module]: Only this point is blocking

File: lib/features/farnet_communities/farnet_communities.module (hunk 2 above)
Comment (bianchim) [inline lib/features/farnet_communities/farnet_communities.module]: Indeed the term 'node' is not clear for a user, removing it in both page title.

File: lib/features/farnet_communities/farnet_communities.module (hunk 1 above)
Comment (bianchim) [inline lib/features/farnet_communities/farnet_communities.module]: Added the 'farnet_communities' halfway and forgot this one, fixing it

File: lib/features/farnet_communities/farnet_communities.module (hunk 5 above)
Comment (bianchim) [inline lib/features/farnet_communities/farnet_communities.module]: I initially wanted to show the tab only on the 'view published' page, which was silly. I changed it but forgot to remove it afterwards.

**Pull request merged**